src/main/java/org/opentripplanner/netex/mapping/AuthorityToAgencyMapper.java

```diff
@@ -33,9 +34,21 @@ class AuthorityToAgencyMapper {
    * Map authority and time zone to OTP agency.
    */
   Agency mapAuthorityToAgency(Authority source) {
+    String agencyName = source.getName() != null &&
+      StringUtils.hasValue(source.getName().getValue())
+      ? source.getName().getValue()
+      : null;
+
+    String agencyShortName = source.getShortName() != null &&
+      StringUtils.hasValue(source.getShortName().getValue())
+      ? source.getShortName().getValue()
+      : null;
+
+    String mainName = agencyName != null ? agencyName : agencyShortName;
```

Comment:
```suggestion
    String mainName = StringUtils.hasValue(agencyName) ? agencyName : agencyShortName;
```

Reply:
Please deduplicate even further.

@vpaturet We could also make `MultilingualStringMapper.nullableValueOf` check for a blank string. WDYT?

Reply:
Sounds like a good idea. I don't see any risk of regression.

Reply:
@cambio-deca Can you implement this?

Reply:
Something like this?

  ```
  public static String nullableValueOf(@Nullable MultilingualString string) {
    if (string == null) {
      return null;
    }

    String value = string.getValue();
    if (value == null || value.isBlank()) {
      return null;
    }
    
    return string.getValue();
  }
  ```

Reply:
Yes exactly, but please use `StringUtils.hasValue` or `StringUtils.hasNoValue`.